packages/tamagotchi/src/renderer/index.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
```

Comment:
scale?

Reply:
This renderer html is just a  placeholder currently.

Reply:
Ah I mean this part:

```suggestion
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=0" />
```

Reply:
nvm, already having it.